patches/api/0163-Make-the-default-permission-message-configurable.patch

```diff
@@ -43,15 +43,28 @@ index ca4e2d3b27f629e0d5e672fc915a5d03f0c0581d..17f8dd9870a47227a7c9bb09cceedb94
       * Creates a PlayerProfile for the specified uuid, with name as null
       * @param uuid UUID to create profile for
 diff --git a/src/main/java/org/bukkit/command/Command.java b/src/main/java/org/bukkit/command/Command.java
-index 7c80dc54776d0d66f7816b77136f6dbd9b801704..c10fc8d2386301bc2caddcdb1cd18566bcaa8689 100644
+index 7c80dc54776d0d66f7816b77136f6dbd9b801704..67d72af7b10c7f1d09a742cabe68c595606655e5 100644
 --- a/src/main/java/org/bukkit/command/Command.java
 +++ b/src/main/java/org/bukkit/command/Command.java
-@@ -185,7 +185,7 @@ public abstract class Command {
+@@ -5,6 +5,7 @@ import java.util.ArrayList;
+ import java.util.Collections;
+ import java.util.List;
+ import java.util.Set;
++
```

Comment:
Don't need this newline.

Reply:
Derp, sorry. Fixed.

*I was completely confident when I first wrote this, but we're now on the 3rd requested change on a 3 line change lol. Sorry for wasting everyone's time*